docs/references/react/use-sign-in.mdx

```diff
@@ -53,35 +53,60 @@ The `useSignIn()` hook provides access to the [`SignIn`](/docs/references/javasc
 
 ## How to use the `useSignIn()` hook
 
-### Check the current state of a sign-in with `useSignIn()`
+### Check the current state of a sign-in
 
-Use the `useSignIn()` hook to check the current state of a sign-in.
+Use `useSignIn()` to check the current sign-in status and display the value in the UI.
 
-```tsx {{ filename: 'pages/sign-in-step.tsx' }}
+```tsx {{ filename: 'src/components/SignInPage.tsx' }}
 import { useSignIn } from '@clerk/clerk-react'
 
-export default function SignInStep() {
+export default function SignInPage() {
   const { isLoaded, signIn } = useSignIn()
 
   if (!isLoaded) {
-    // Add logic to handle loading state
+    // Handle loading state
     return null
   }
 
-  return <div>The current sign in attempt status is {signIn.status}.</div>
+  return <div>The current sign in attempt status is {signIn?.status}.</div>
 }
 ```
 
+### `SignIn` status values
+
 The `status` property of the `SignIn` object can be one of the following values:
 
-| Values | Descriptiom |
-| - | - |
-| `complete` | The user has been signed in and custom flow can proceed to `setActive()` to create session. |
-| `needs_first_factor` | The first factor verification is missing. One of `email_link`, `email_code`, `phone_code`, `web3_metamask_signature`, `web3_coinbase_wallet_signature`, `web3_okx_wallet_signature` or `oauth_provider` is required to verify the user. See [the reference](/docs/references/javascript/sign-in/first-factor) for more information. |
-| `needs_second_factor` | The Second Factor verification is missing. The Second Factor is an optional step to provide additional verification and includes `phone_code` and `totp`. See [Second Factor](/docs/references/javascript/sign-in/second-factor) for details. |
-| `needs_identifier` | The user's identifier (email address, phone number, username, etc.) hasn't been provided. |
-| `needs_new_password` | The user needs to set a new password. |
+<Properties>
+  - `complete`
+  - `string`
+
+  The user has been signed in and the custom flow can proceed to `setActive()` to create a session.
+
+  ---
+
+  - `needs_first_factor`
+  - `string`
+
+  One of the following [first factor verification strategies](/docs/references/javascript/sign-in/first-factor) is missing: `email_link`, `email_code`, `phone_code`, `web3_metamask_signature`, `web3_coinbase_wallet_signature` or `oauth_provider`.
+
+  ---
+
+  - `needs_second_factor`
+  - `string`
+
+  One of the following [second factor verification strategies](/docs/references/javascript/sign-in/second-factor) is missing: `phone_code` or `totp`. This value is optional.
+
+  ---
+
+  - `needs_identifier`
+  - `string`
 
-### Create a custom sign-in flow with `useSignIn()`
```

Comment:
these sections need to stay - they were added a result of user feedback.
we can rename to "Example" if that makes more sense

Reply:
https://github.com/clerk/clerk-docs/pull/1814/commits/70af8fedfe4a526a54f434837cf77c6e25921384